rizinorg/cutter
# Use rizin instead of Qt to parse JSON

**Your checklist for this pull request**


**Detailed description**

Qt truncates integers to 52-bit, corrupting e.g. memory addresses. Use rizin's JSON parser, which can parse integers whose size is up to 64 bits.

**Test plan (required)**

1. Install Kernel Debug Kit 12.2 build 21D49. For details, see: https://developer.apple.com/documentation/apple-silicon/debugging-a-custom-kernel-extension
2. Open `/Library/Developer/KDKs/KDK_12.2_21D49.kdk/System/Library/Kernels/kernel.development.t8101`. 
3. See it can disassemble 0xfffffe0007cc7290.

It changes everywhere using JSON so please review the code carefully and test anywhere suspicious.

**Closing issues**
